app/views/layouts/application.html.erb

```diff
@@ -92,6 +92,7 @@
     </div>
   <% end %>
   <div id="page-content-inner">
+    <div id="page-route-change" class="screen-reader-only" aria-live="assertive" aria-atomic="true"></div>
```

Comment:
```suggestion
    <div id="page-route-change" class="screen-reader-only" aria-live="polite" aria-atomic="true"></div>
```
nitpick (blocking) - just a small tweak but we can make this polite so it doesn't interrupt any other important announcements 🙂